Makefile

```diff
@@ -41,10 +41,19 @@ isort-check:
 test:
 	pytest
 
+## - test-cov
+##	Run pytest tests with coverage report.
+.PHONY: test-cov
+test-cov:
+	pytest --cov=poglink --cov-report=xml:coverage.xml
+	@coverage report
```

Comment:
`coverage report` just takes the generated `.coverage` file and renders a report in stdout. `coverage html` will render it into html pages found in the `htmlcov/` dir. You can open `htmlcov/index.html` in your browser to easily browse the report.

`coverage xml` is implicitly called via the last `pytest` arg.